open-metadata-resources/open-metadata-labs/administration-labs/understanding-platform-services.ipynb

```diff
@@ -521,7 +521,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.9.7"
```

Comment:
this Python upgrade appears to be the main change here. Can you confirm that the labs still run at this later Python version

Reply:
This is what I tested, so yes. Labs still works with this python version

Reply:
Just found out why the version is higher.
In the docker build for jupyter we always retrieve the latest version

FROM docker.io/jupyter/base-notebook:latest

Reply:
Yes - we do. It's a fair question as to whether we should pin, and then upgrade similarly to dependabot, but with limited time it (and in fact nginx) are being allowed to float up to latest.